pallets/parachain-staking/src/lib.rs

```diff
@@ -2508,11 +2626,99 @@ pub mod pallet {
+
+            let vote = Self::get_vote(growth_id);
+            ensure!(Self::can_end_vote(&vote), Error::<T>::ErrorEndingVotingPeriod);
+
+            let growth_is_approved = vote.is_approved();
+
+            let growth_info = Self::try_get_growth_data(&growth_id.period)?;
+            let rewards_in_period_128 = TryInto::<u128>::try_into(growth_info.total_staker_reward)
+                .map_err(|_| Error::<T>::ErrorConvertingBalance)?;
+
+            let average_staked_in_period_128 = TryInto::<u128>::try_into(growth_info.total_stake_accumulated / growth_info.number_of_accumulations.into())
+                .map_err(|_| Error::<T>::ErrorConvertingBalance)?;
+
+            if growth_is_approved {
+                if growth_info.tx_id.is_some() {
+                    let result =
+                        T::CandidateTransactionSubmitter::submit_candidate_transaction_to_tier1(
+                            EthTransactionType::TriggerGrowth(TriggerGrowthData::new(
+                                rewards_in_period_128,
+                                average_staked_in_period_128,
+                                growth_id.period 
+                            )),
+                            *growth_info.tx_id.as_ref().expect("checked on the IF statement"),
+                            growth_info.added_by.ok_or(Error::<T>::GrowthTxSenderNotFound)?,
+                            voting_session.state()?.confirmations,
+                        );
+
+                    if let Err(result) = result {
+                        log::error!("❌ Error Submitting Tx: {:?}", result);
+                        Err(result)?
+                    }
+                    // There are a couple possible reasons for failure.
+                    // 1. We fail before sending to T1: likely a bug on our part
+                    // 2. Quorum mismatch. There is no guarantee that between accepting a growth and
+                    // submitting it to T1, the tier2 session hasn't changed and with it
+                    // the quorum, making ethereum-transactions reject it
+                    // In either case, we should not slash anyone.
+                }
+                // If we get here, then we did not get an error when submitting to T1.
+
+                // create_and_report_summary_offence::<T>(
+                //     &reporter,
+                //     &vote.nays,
+                //     SummaryOffenceType::RejectedValidRoot,
+                // );
+                
+                <Growth<T>>::mutate(growth_id.period, |growth| {
+                    growth.triggered = Some(true)
+                });
+                
+                Self::deposit_event(Event::<T>::TriggeringGrowth {
+                    growth_period: growth_id.period,                    
+                });
+            } else {
+                // We didn't get enough votes to approve this growth
+
+                let growth_creator =
```

Comment:
Is this needed at the moment? It does not seem used anywhere.

Reply:
No its not, I have another PR that cleans up unused references but will also remove it here.